cips/cip-17.md

```diff
@@ -4,7 +4,8 @@ title: Lemongrass Hardfork
 description: Reference specifications included in the Lemongrass Hardfork
 author: Evan Forbes (@evan-forbes)
 discussions-to: https://forum.celestia.org/t/lemongrass-hardfork/1589
-status: Draft
```

Comment:
is it possible to move from draft -> last call? or should I move from draft -> review for some time (i.e. a week) -> last call with a 2 week deadline?

Reply:
it isn't abundantly clear, but it is clear enough that it probably _should_ follow each step https://github.com/celestiaorg/CIPs/blob/main/cips/cip-1.md#cip-process

Reply:
In the CIP call today, Yaz said it was fine to go straight to last call

Reply:
sgtm